tests/aws/scenario/test_ecs_scenario.py

```diff
@@ -50,3 +52,56 @@ def test_scenario_call_service(self, aws_client, infrastructure):
         # TODO: add a test here to call the deployed NGINX service
         # outputs = infrastructure.get_stack_outputs(stack_name="ClusterStack")
         requests.get("...")  # not working yet
+
+
+
+class TestEcsScenario2:
+    @markers.aws.unknown
+    def test_scenario_ecs(self, aws_client):
+        vpc = aws_client.ec2.create_vpc(CidrBlock="0.0.0.0/16")
```

Comment:
if this is only a leftover, it should probably be removed :) else we are definitely missing cleanups :)